docs/pages/access-controls/access-requests/access-request-configuration.mdx

```diff
@@ -189,6 +189,15 @@ privileges:
    earlier, whichever is shorter. Otherwise, set the duration of elevated
    privileges to the session TTL.
 
+### Setting when users can assume elevated privileges
+
+When creating or reviewing Access Requests, you can specify the earliest time
```

Comment:
I wasn't a reviewer on the original PR but we should also mention in this section what is the effect of this flag when reviewing requests. Specifically, the fact that as a requester you can request a specific start time but reviewers can override it when approving the request, and that currently that last reviewer to change it "wins".

This should also mention that it can't be set in the past.

Reply:
Made a PR to add the extra details to master [here](https://github.com/gravitational/teleport/pull/36401)